domains/admin-portal/AdminPortal.Web/AdminPortal.Tests/Controllers/WhitelistedOrganizationsControllerTests.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using AdminPortal._Features_;
+using AdminPortal.Dtos.Response;
+using AdminPortal.Tests.Setup;
+using Microsoft.AspNetCore.Mvc.Testing;
+using NSubstitute;
+
+namespace AdminPortal.Tests.Controllers;
+
+public class WhitelistedOrganizationsControllerTests
```

Comment:
Instead of spinning a test factory up here, would it make sense to just test the contract between authorization and admin portal?
We can have these test in authorization subsystem